pkg/aks/check.go

```diff
@@ -113,7 +113,7 @@ var regionToOmsRegionMap = map[string]string{
 	"chinanorth2": "chinaeast2",
 }
 
-func CheckLogAnalyticsWorkspaceForMonitoring(ctx context.Context, client services.WorkplacesClientInterface,
+func checkLogAnalyticsWorkspaceForMonitoring(ctx context.Context, client services.WorkplacesClientInterface,
```

Comment:
`CheckLogAnalyticsWorkspaceForMonitoring` is public function, I was thinking that we want keep them starting with UpperCase

Reply:
I noticed it's only used inside `aks` package so I thought we can make it private

Reply:
I've just done a quick search across rancher org https://github.com/search?q=org%3Arancher+CheckLogAnalyticsWorkspaceForMonitoring&type=code
it should be safe to make it private

Reply:
Could we consider adding apidiff in the future to check for public api changes?

Reply:
Yes, I can add apidiff check